Sources/Plasma/CoreLib/hsMatrix44.cpp

```diff
@@ -115,9 +115,9 @@ hsMatrix44 hsMatrix44::mult_accelerate(const hsMatrix44 &a, const hsMatrix44 &b)
         return b;
     if( b.fFlags & hsMatrix44::kIsIdent )
         return a;
-    
-    vDSP_mmul((const float*)a.fMap, 1, (const float*)b.fMap, 1, (float*)&(c.fMap), 1, 4, 4, 4);
-    
+
+    vDSP_mmul((float*)a.fMap, 1, (float*)b.fMap, 1, (float*)&(c.fMap), 1, 4, 4, 4);
```

Comment:
Would this work to make it clear that we mean to cast away the `const` here? (Replacing `fMap` with `fMap[0]` should make the other part of the cast unnecessary, because `fMap[0]` will decay to a `float*`.)

```suggestion
    vDSP_mmul(const_cast<float*>(a.fMap[0]), 1, const_cast<float*>(b.fMap[0]), 1, const_cast<float*>(c.fMap[0]), 1, 4, 4, 4);
```

Reply:
The `fMap` property in hsMatrix44 is not `const`, so I don't think there's any issue here. It's fine to pass a `float*` to a function that's expecting `const float*` (the `const` just means that the function guarantees it won't modify the parameter).

`fMap[0]` might be better than casting though... let me try that

Reply:
`a` and `b` are `const` references though, so their `fMap`s will be `const` I think - it's an array field, not a pointer, so the `const`-ness of the entire object will "propagate" to it. `c.fMap` won't need a cast though, as you say.

Reply:
Ahh, good catch there, I've updated the PR